<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Covek samo srcem dobro vidi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 17, 26);
color:rgb(57,255,20)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>color filter:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbColor">
           <item>
            <property name="text">
             <string>red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>superimposed</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pbShowLayer">
         <property name="text">
          <string>show layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbExif">
         <property name="text">
          <string>exif</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbZstag">
         <property name="text">
          <string>zstag</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbString">
         <property name="text">
          <string>string</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSteghide">
         <property name="text">
          <string>steghide</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGraphicsView" name="imgView"/>
       </item>
       <item>
        <widget class="QWidget" name="wHidden" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>549</width>
            <height>37</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="hidden">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignLeft">
            <widget class="QPushButton" name="pbPrevious">
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QPushButton" name="pbNext">
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbUploadImage">
         <property name="text">
          <string>upload image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
